docs/modules/tiered-storage/pages/configuration.adoc

```diff
@@ -51,21 +53,21 @@ XML::
         <in-memory-format>NATIVE</in-memory-format> <3>
         <tiered-store enabled="true"> <4>
             <memory-tier>
-                <capacity>256 MB</capacity> <5>
+                <capacity unit="MEGABYTES" value="256"/> <5>
             </memory-tier>
             <disk-tier enabled="true" device-name="my-disk"/> <6> <7>
         </tiered-store>
     </map>
 </hazelcast>
 ----
 ====
-<1> Name of the disk; `my-disk` in this example
-<2> Name of the directory where the data on disk is stored
-<3> The in-memory format for the map having Tiered Storage enabled MUST be `NATIVE`
-<4> Tiered Storage is disabled by default; set this parameter `true` to enable it
-<5> Capacity of the memory in which the frequently accessed data will be stored; cannot be set to `0`
-<6> Whether to use the device (disk) as storage
-<7> Name of the device (disk) that you set under the `local-device` configuration
+<1> Name of the disk; `my-disk` in this example.
+<2> Name of the directory where the data on disk is stored.
```

Comment:
maybe "The directory to be used for storing device data. Can be an absolute or relative path to the node startup directory." ?